week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,26 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+<<<<<<< Updated upstream
+return "Hello, my name is " + name + " and I am " + age + " years old";
+=======
+return (`Hello, my name is ${name} and I am ${age} years old`);
+>>>>>>> Stashed changes
+}
```

Comment:
I think you had some git conflicts when you unstashed your changes here and below 😅